.gitignore

```diff
@@ -1,3 +1,7 @@
+# Auto generated files for testing
+test/fixtures/templates/bad/limit_numbers.yaml
+test/fixtures/templates/bad/limit_size.yaml
```

Comment:
good catch 👍  should've added this in https://github.com/aws-cloudformation/cfn-python-lint/pull/1747 but too accustomed to testing remotely with Travis